cmd/otelcol/Dockerfile

```diff
@@ -35,6 +35,12 @@ RUN if [[ "$ARCH" = "amd64" || "$ARCH" = "arm64" ]]; then \
         mkdir -p /usr/lib/splunk-otel-collector/agent-bundle; \
     fi
 
+FROM alpine:3.17.0 AS jmx
+ARG JMX_METRIC_GATHERER_RELEASE
+RUN JMX_METRICS_JAR=opentelemetry-jmx-metrics.jar && \
```

Comment:
Should the jar name match the [default path](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/jmxreceiver#jar_path-default-optopentelemetry-java-contrib-jmx-metricsjar), i.e. `opentelemetry-java-contrib-jmx-metrics.jar`, or will it be overridden?

Reply:
You're right, it should match the default path. I renamed the paths to match the file being downloaded, forgetting the actual receiver's requirement here. Thanks!